src/Model/ObjectsLoader.php

```diff
@@ -191,7 +192,9 @@ protected function loadSingle(int $primaryKey, ObjectType $objectType, ?array $o
         /** @var \BEdita\Core\Model\Entity\ObjectEntity $object */
         $object = $action(compact('primaryKey', 'lang', 'contain'));
 
-        return $this->autoHydrateAssociations($this->setJoinData([$object], $contain), $depth, $hydrate)->first();
+        return $this->autoHydrateAssociations($this->setJoinData([$object], $contain), $depth, $hydrate)
```

Comment:
Ma quindi, in automatico, se è settata una traduzione e carichiamo il LocaleMiddleware, **ogni** caricamento di oggetto ritorna l'oggetto tradotto?

Ha senso passare un parametro, se vogliamo ignorare la traduzione puntualmente, alla loadSingle e loadMulti?

Reply:
Potremmo in effetti prendere il `lang` dalle options invece di usare il trait i18n di BEdita, e spostare la [magia](https://github.com/chialab/bedita-frontend-kit/discussions/46#discussioncomment-2698782) nelle generic actions. @fquffio?

Reply:
Mmh, oddio forse sarebbe più bellino. Oppure ci teniamo la configurazione qui, ma diamo un minimo di opzioni:

Tipo, potremmo avere una conf del tipo `lang: string|null|false`:
- se `string`, traduce nel linguaggio richiesto
- se `null` (default), prende il linguaggio di default da `$this->getLang()`, ovvero con la magia oscura che c'è ora
- se `false`, non traduce nulla e dà sempre l'oggetto originale